### PR TITLE
Adding new repository for indexing: amcl3d

### PR DESCRIPTION
**Pull request by palomacf-devs**
Adaptative Monte Carlo Localization (AMCL) in 3D.
Amcl3d is a probabilistic algorithm to localizate a robot moving in 3D.
It use a Monte-Carlo method, a particle filter. 
This package use a laser sensor and range sensor to localizate the UAV in a know map.

### CODE REVIEW

**Review by tfoote — CHANGES_REQUESTED** [review]
This repository appears to be empty

Screenshot from 2019-08-26 16-53-12

Please either release the code there or close this PR.

**Comment by pcarrascof** [comment]
The code has been released on GitHub!

**Review by clalancette — CHANGES_REQUESTED** [review]
Needs two changes to pass CI.

File: kinetic/distribution.yaml [hunk 1]
```diff
@@ -191,6 +191,12 @@ repositories:
       url: https://github.com/RobotnikAutomation/agvs_sim.git
       version: kinetic-devel
     status: maintained
+  amcl3d:
+    doc:
+      type: git
+      url: https://github.com/fada-catec/amcl3d
+      version: 1.0
```
[the remark this answers is not in the document]
Comment (pcarrascof) [reply kinetic/distribution.yaml]: It has been changed

File: kinetic/distribution.yaml [hunk 2]
```diff
@@ -191,6 +191,12 @@ repositories:
       url: https://github.com/RobotnikAutomation/agvs_sim.git
       version: kinetic-devel
     status: maintained
+  amcl3d:
+    doc:
+      type: git
+      url: https://github.com/fada-catec/amcl3d.git
```
[the remark this answers is not in the document]
Comment (pcarrascof) [reply kinetic/distribution.yaml]: It's done!

**Pull request merged**